cfpb/capital-framework
# Update to webpack 4

## Additions

- Add webpack-config file.

## Changes

- Update `webpack` to `4.8.3`, `webpack-stream` to `4.0.3`, `uglifyjs-webpack-plugin` to `1.2.5`, `babel-core` to `6.26.3`, and `babel-preset-env` to `1.7.0`.

## Testing

1. run `npm install`
1. run `gulp`, should build and pass tests.
2. `/dist/capital-framework.min.js`, `/tmp/cf-expandables/cf-expandables.min.js`, and `/tmp/cf-tables/cf-tables.min.js` should be minified.
